resources/views/admin/users/allusers.blade.php

```diff
@@ -45,52 +45,56 @@
                         {{-- We will change it later to match the actual user table. --}}
                         @foreach($all_users as $user)
-                        <tr class="usertable-tr">
-                            <td>{{$user->id}}</td>
-                            <td>
-                                @if($user->avatar)
-                                    <img src="#" alt="{{$user->avatar}}">
-                                @else
-                                    <i class="fa-solid fa-circle-user avatar-lg"></i>
-                                @endif
-                            </td>
-                            <td>{{$user->username}}</td>
-                            <td>{{$user->fullname}}</td>
-                            <td>{{$user->email}}</td>
-                            <td>{{$user->created_at}}</td>
-                            <td>{{$user->updated_at}}</td>
-                            <td>
-                                <div class="dropdown">
-                                    <button class="btn" data-bs-toggle="dropdown">
-                                        <i class="fa-solid fa-circle text-success"></i>
-                                    </button>
-                                   <div class="dropdown-menu">
-                                        <a href="#" class="dropdown-item">
-                                            <i class="fa-solid fa-circle text-success"></i> activate
-                                        </a>
-                                        <a href="#" class="dropdown-item">
-                                            <i class="fa-solid fa-circle text-danger"></i> diactivate
-                                        </a>
-                                        <a href="#" class="dropdown-item">
-                                            <i class="fa-solid fa-circle text-dark"></i> deleted
-                                        </a>
-                                   </div>
-                                </div>
-                            </td>
-                            <td>
-                                <div class="dropdown">
-                                    <button class="btn" data-bs-toggle="dropdown">
-                                        <i class="fa-solid fa-ellipsis"></i>
-                                    </button>
-                                   <div class="dropdown-menu">
-                                       <button class="dropdown-item" data-bs-toggle="modal" data-bs-target="#show-profile-{{$user->id}}">
-                                        show profile
-                                       </button>
-                                   </div>
-                                   {{-- @include('admin.users.modal.showprofile') --}}
-                                </div>
-                            </td>
-                        </tr>
+                            @if ($user->user_name === 'The Admin')
```

Comment:
but if I remove that part, it shows the admin in users table too.
I don't want to show it. could you give me anything else how to solve my issue?

Reply:
instead of having
```
@if ($user->user_name === 'The Admin')
{{-- the admin won't show users table --}}
@else
```
you can just say @if($user->name != 'The Admin')
but still that's not the correct implementation because we don't wan't the hard code everything here
for example we have multiple users that are admin, we don't want to create multiple `if` statement for that.
what I want you to do is just base it on the user roles something like @if($user->role == 'admin')